### PR TITLE
Use `http.get` built-in cache rather than cache.star

### PR DESCRIPTION
**Pull request by rohansingh**
Pass `ttl_seconds` to `http_get` and let it handle caching, rather than using `cache.star` explicitly. This is the preferred practice and is generally more efficient.

### CODE REVIEW

**Comment by rohansingh** [comment]
✨✨ Here's [an AI-assisted sketch of how you might approach this issue](https://copilot-workspace.githubnext.com/tidbyt/community/pull/2546?shareId=e850c88f-b943-4763-a15a-293cd8b0b6eb) saved by @rohansingh using Copilot Workspace v0.22

<details open><summary><b>Topic</b></summary>
<br/>

Remove apps/pslcricket entirely, since it doesn't work

</details>

<details open><summary><b>Before</b></summary>
<br/>

* `apps/pslcricket/manifest.yaml` and `apps/pslcricket/psl_cricket.star` still exist in the repository.
* The `pslcricket` app is still referenced in the `runtime_exceptions` array in the `.github/scripts/check-apps.sh` file.
* The `pslcricket` app is not removed from the repository as per the task requirement.



</details>
<details open><summary><b>After</b></summary>
<br/>

* `apps/pslcricket/manifest.yaml` and `apps/pslcricket/psl_cricket.star` have been deleted from the repository.
* The `pslcricket` app has been removed from the `runtime_exceptions` array in the `.github/scripts/check-apps.sh` file.


</details>

<details open><summary><b>Plan</b></summary>
<br/>

* `apps/pslcricket/manifest.yaml` (DELETE)
* `apps/pslcricket/psl_cricket.star` (DELETE)
* `.github/scripts/check-apps.sh` (CHANGE)
  - Remove the `pslcricket` app from the `runtime_exceptions` array

</details>

<details open><summary><b>Sketch of implementation</b></summary>
<br/>

[View the changes](https://copilot-workspace.githubnext.com/tidbyt/community/pull/2546?shareId=e850c88f-b943-4763-a15a-293cd8b0b6eb)

<details open><summary><b>Details</b></summary>
<br/>

Code analyzed at 4ca7716fa14c25e39d857c065767316df9d5d7ce

</details>

**Comment by rohansingh** [comment]
Remove the `pslcricket` app from the repository

* Delete `apps/pslcricket/manifest.yaml`
* Delete `apps/pslcricket/psl_cricket.star`
* Remove the `pslcricket` app from the `runtime_exceptions` array in `.github/scripts/check-apps.sh`


---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/tidbyt/community/pull/2546?shareId=7906fa38-538f-4336-acfe-b3a633075a2e).

File: apps/cpblscores/cpbl_scores.star [hunk 1]
```diff
@@ -116,7 +114,7 @@ def main(config):
                 if NScount > NSMax:
                     continue
             gameTime = s.get("strTimestamp")
-            convertedTime = time.parse_time(gameTime.replace(":00+00:00", "Z"), format = "2006-01-02T15:04Z").in_location(timezone)
```
Comment (matslina) [inline apps/cpblscores/cpbl_scores.star:119]: this is intentional?

Comment (rohansingh) [reply apps/cpblscores/cpbl_scores.star:119]: Yes. Looks like the format returned by the API changed or something, but this wasn't working.

**Pull request merged**